cypress/integration/login.js

```diff
@@ -12,16 +12,20 @@ describe('Sourcebooks login', function() {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Ugne Lekstutiene"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
         cy.get('[aria-label="User"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
+        cy.get('.calendar--today').contains(new Date().getDate());

```

Comment:
It is best to move this after line: 
cy.get('.calendar').should('be.visible')
cypress is flexible on this one and not failing, but other frameworks might fail because it is not yet loaded